lib/v2/qweather/3days.js

```diff
@@ -13,34 +16,64 @@ module.exports = async (ctx) => {
         }
         return data[0].id;
```

Comment:
```suggestion
        return response.data.location[0].id;
```
No need to form an array at all since the `location[0]` is always used.